src/SignalRServiceExtension/Bindings/LiteServiceEndpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class LiteServiceEndpoint
+    {
+        public string ConnectionString { get; set; }
```

Comment:
Is the `ConnectionString` property still needed here? It seems Endpoint/Name/Type/Online should be enough for routing.